hoodie-spark/src/test/scala/com/uber/hoodie/HoodieSparkSqlWriterSuite.scala

```diff
@@ -38,4 +41,12 @@ class HoodieSparkSqlWriterSuite extends FunSuite with Matchers {
     }
   }
 
+  test("throws hoodie exception than no such element") {
```

Comment:
Can you rename this test to "throw hoodie exception when invalid serializer"

Reply:
ok